rtos/TARGET_CORTEX/rtx5/RTX/Source/TOOLCHAIN_ARM/TARGET_M23/irq_armv8mbl.S

```diff
@@ -79,7 +87,7 @@ SVC_Context
                 CBZ      R1,SVC_ContextSwitch   ; Branch if running thread is deleted
 
 SVC_ContextSave
-                IF       __DOMAIN_NS = 1
+                IF       DOMAIN_NS = 1
```

Comment:
Flag name is changed by CMSIS, will have to check Mbed OS and change the flag name everywhere.

Reply:
Sorry for that inconvenience. The double underscore prefix is "reserved" for compiler intrinsic. Hence we removed this prefix to clarify that `DOMAIN_NS` is a user define and not a compiler one.

There are two "versions" of the IRQ assembly file for Armv8-M: irq_armv8mbl.s and irq_armv8mbl_ns.s. The former one contains the actual implementation. The second one just sets this define and includes the former one. There should be no need to know about this define anywhere else than locally within those two files.